app/js/view/list_view.js

```diff
@@ -1,4 +1,4 @@
-import { View } from 'components/fxos-mvc/dist/mvc';
+import /* global Sanitizer */ { View } from 'components/fxos-mvc/dist/mvc';
```

Comment:
Hm.. Now that I see this in context, I actually think this `/* global Sanitizer */` declaration should probably be on a separate line. If you were declaring `View` as global here, then I think this would be ok.